docs/tutorials/getting_started.md

```diff
@@ -1,12 +1,16 @@
 # Getting started
 
-In this tutorial, we will deploy and run the SD-Core 5G core network using Juju. We will also
-deploy a radio and cellphone simulator to simulate usage of this network.
+In this tutorial, we will deploy and run the SD-Core 5G core network using Juju and Terraform.
+As part of this tutorial, we will also deploy additional components (gNB Simulator - a 5G radio 
+and a cellphone simulator, SD-Core Router - a software router facilitating communication between
+the core and the Radio Access Network (RAN)) to simulate usage of this network. Both gNB Simulator 
+and SD-Core Router serve only demonstration purposes and shouldn't be part of production 
```

Comment:
```suggestion
and SD-Core Router serve only for demonstration purposes and shouldn't be part of production 
```

Reply:
I think my version was ok: https://dictionary.cambridge.org/dictionary/english/serve-a-purpose

Reply:
As the SD-Core Router is a singular object, the verb needs to be `serves`